bayesian.go

```diff
@@ -304,6 +410,9 @@ func (c *Classifier) LogScores(document []string) (scores []float64, inx int, st
 // may or may not be a concern. Consider using SafeProbScores()
 // instead.
 func (c *Classifier) ProbScores(doc []string) (scores []float64, inx int, strict bool) {
+	if c.tfIdf && !c.didConvertTfIdf {
+		panic("Error:Using TF-IDF classifier:Need to call ConvertTermsFreqsToTFIDF's first ")
```

Comment:
Change error message to "Using a TF-IDF classifier. Please call ConvertTermsFreqToTfIdf before calling ProbScores."